.github/workflows/test.yml

```diff
@@ -37,3 +38,14 @@ jobs:
         run: |
           forge test -vvv
         id: test
+
+      - name: Run Forge docs
+        run: |
+          forge doc --out docs/docs/contracts
+          python3 docs_reformat_scripts/forge_doc_reformat.py
```

Comment:
How about putting this into a `/docs/scripts/` instead of creating a top level folder?